pkg/os/disk/disk.go

```diff
@@ -0,0 +1,30 @@
+package disk
+
+import (
+	"encoding/json"
+	"fmt"
+	"os/exec"
+)
+
+type DiskInfo struct {
+	Format      string `json:"format"`
+	BackingFile string `json:"backing-filename"`
+	ActualSize  int64  `json:"actual-size"`
+	VirtualSize int64  `json:"virtual-size"`
+}
+
+func GetDiskInfo(imagePath string) (*DiskInfo, error) {
+	// #nosec No risk for attacket injection. Only get information about an image
```

Comment:
```suggestion
	// #nosec No risk for attacker injection. Only get information about an image
```
Could/should we use safepath?

Reply:
Well, that's a good question. From what I've seen in other places in virt-launcher, we never use safepath when calling `GetDiskInfo`, so I guess not.